src/libs/ReactNativeLauncher.js

```diff
@@ -309,6 +309,7 @@ class ReactNativeLauncher extends Launcher {
       timerId = setTimeout(() => {
         reject(ERRORS.SET_WORKER_STATE_TOO_LONG_TO_INIT)
       }, options?.timeout || SET_WORKER_STATE_TIMEOUT_MS)
+      this.emit('SET_WORKER_STATE', options)
```

Comment:
I don't understand why this change is necessary in this commit? Can you explain?

Reply:
Je vais mieux y arriver en français : l'évènement `SET_WORKER_STATE` est ce qui provoque potentiellement le rechargement de la webview et donc l'évènement `worker:webview:ready`. Avant ce changement, on écoutait l'évènement `worker:webview:ready` après avoir déclenché `SET_WORKER_STATE`, il y avait donc un risque (minime) de rater `worker:webview:ready` et de rester bloqué.